packages/wallets/src/features/accounts/screens/ResubmitPOA/ResubmitPOA.scss

```diff
@@ -0,0 +1,83 @@
+.wallets-resubmit-poa {
+    width: 99.6rem;
+    height: 60rem;
+    display: flex;
+    padding: 1.6rem;
+    flex-direction: column;
+    align-items: center;
+    gap: 1.6rem;
+    flex: 1 0 0;
+    align-self: stretch;
+    background: var(--system-light-8-primary-background, #fff);
+
+    @include mobile {
+        width: 100%;
+        height: 100%;
+        padding: 1.6rem;
+    }
+
+    &__address {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-start;
+        gap: 1.6rem;
+        max-width: 85rem;
+
+        @include mobile {
+            width: 100%;
+        }
+
+        &__title {
+            display: flex;
+            height: 2.4rem;
+            justify-content: center;
+            align-items: center;
+            gap: 1.1rem;
+
+            @include mobile {
+                display: flex;
+                align-items: center;
+                gap: 8px;
+                align-self: stretch;
+            }
+
+            &--divider {
```

Comment:
is that a block or modifier?

Reply:
from the .jsx code it looks like a static element, like a block,

example of blocks:
__message
__title
__header

example of modifiers:
--mobile
--large
--primary

Reply:
But address itself its a block, and everything inside are modifiers right

Reply:
hmm, block is more like a section or paragraph, 
e.g. lets consider .css describing skyscrapper

.skyscrapper {
 &__roof {
  // block (element of skyscrapper)
 }
 
 &__basement {
 // block (element of skyscrapper)
 }
 
 &__level {
 // block (element of skyscrapper)
 }

 &--shiny {
 // modifier - adjective, describing styles for shiny skyscrappers, could be e.g. color: silver etc
 }
 
 &--tall {
 // modifier - describing styles for tall skyscrappers, could be e.g. height: 800 meters;
 }
 
 &--short {
 // modifiers - describing styles for short skuyscrappers, could be e.g. height: 50 meters;
 }
}